src/store/modules/getters/staticData.js

```diff
@@ -22,3 +24,14 @@ export const getNFTClassMintedCount = state => id =>
     (acc, val) => acc + val.length,
     0
   );
+
+export const getNFTClassIdSorter = (_, getters) => classIds => {
+  const sorted = [...classIds].sort((a, b) => {
+    const aIsWritingNFT = isWritingNFT(getters.getNFTClassMetadataById(a));
+    const bIsWritingNFT = isWritingNFT(getters.getNFTClassMetadataById(b));
+    if (aIsWritingNFT && !bIsWritingNFT) return -1;
+    if (!aIsWritingNFT && bIsWritingNFT) return 1;
+    return 0;
```

Comment:
Should sort by created time if return 0?

Reply:
Temporary sort by price for now.